misc/python/materialize/mzcompose/services.py

```diff
@@ -100,6 +100,11 @@ def __init__(
                 "persist_sink_minimum_batch_updates=128",
                 "enable_multi_worker_storage_persist_sink=true",
                 "storage_persist_sink_minimum_batch_updates=100",
+                "enable_monotonic_oneshot_selects=true",
+                "persist_pubsub_push_diff_enabled=true",
+                "persist_pubsub_client_enabled=true",
+                "persist_stats_filter_enabled=true",
+                "persist_stats_collection_enabled=true",
```

Comment:
Isn't this the feature that should not be configured to a value higher than 99% because we don't want to guarantee that all stats are recorded? Or do I mistake it for something else? Though, this might not be relevant as long as we don't verify that all stats are there.

Reply:
> When the code determines this is the case, with a N% chance we will fetch the part anyways; if it turns out that any of the records in the part aren’t filtered out, the code will panic.

This sounds to me like it makes sense to set to 100%, which will do filters, but still fetch and compare if the filter was wrong, and then panic.

Reply:
Thanks for the pointers, I will take a separate look at this as I work on explicit pushdown testing.